docker/nginx.conf

```diff
@@ -0,0 +1,7 @@
+server {
```

Comment:
Is there anything about this that you should document? (In a small comment)

Reply:
I don't know, is there? To me it pretty much speaks for itself: we listen on port 80, then act as a proxy and pass on requests for `/` to whatever's listening on port 8000. Maybe there could be a [link to the nginx docs](http://nginx.org/en/docs/beginners_guide.html), but it's the first result in Google if I search for `nginx configuration`, so...

Reply:
Check 👍